requirements/edx/base.txt

```diff
@@ -156,3 +156,6 @@ analytics-python==0.4.4
 # Needed for mailchimp(mailing djangoapp)
 mailsnake==1.6.2
 jsonfield==1.0.3
+
+# Needed for credit notifications
+pynliner==0.5.2
```

Comment:
@feanil @singingwolfboy FYI: this is a dependency that we want to add for inlining CSS into HTML email notifications.